### ISSUE TITLE
Drawing Classifier does not show validation loss while training 

### ISSUE THREAD

**Issue by shreyajain17** [apple/turicreate#1907]
```
+------------------------+------------------------+------------------------+------------------------+------------------------+
| Iteration              | Training Loss          | Training Accuracy      | Validation Accuracy    | Elapsed Time (seconds) |
+------------------------+------------------------+------------------------+------------------------+------------------------+
```

**Comment by srikris** [comment]
@shreyajain17 @shantanuchhabra I'm tempted to say that here, the loss can be removed from the display. I don't think it adds much value.

Thoughts?

**Comment by shreyajain17** [comment]
Can I ask why do you think it doesn't add much value?

**Comment by TobyRoseman** [comment]
The reason I don't think it adds value is that it's hard to interpret. I would be in favor of removing it from the progress table. I also think it should be removed from the `summary()` and `__str__()` methods.

**Comment by shreyajain17** [comment]
Should I remove them then?

**Comment by nickjong** [comment]
Yeah, I'm also in favor of removing loss here

**Comment by shreyajain17** [comment]
So from the conversation till now. It seems like we want to remove training loss, right?
@srikris @nickjong @shantanuchhabra @TobyRoseman
